measures/ResStockArguments/measure.rb

```diff
@@ -408,33 +408,43 @@ def run(model, runner, user_arguments)
 
     # Conditioned floor area
     if args[:geometry_unit_cfa] == Constants.Auto
+      # FIXME: Need to disaggregate detached and mobile home
       cfas = { ['0-499', HPXML::ResidentialTypeSFD] => 298, # AHS 2021, 1 detached and mobile home weighted average
-               ['0-499', HPXML::ResidentialTypeSFA] => 273, # AHS 2021, 1 detached and mobile home weighted average
+               ['0-499', HPXML::ResidentialTypeSFA] => 273, # AHS 2021, 1 attached
```

Comment:
@joseph-robertson Do you know how to generate new numbers that disaggregate SFD and MH?

Reply:
I do not; I believe @afontani did this. But note that we very recently have been discussing modifying this approach with not only a replacement of AHS with RECS, but also some kind of random selection of CFAs within bins. We should probably discuss.

@rajeee

Reply:
In that case, we can probably keep this PR as is. It makes no substantive changes (no changes to simulation results); it only updates the building type in the HPXML and lays the groundwork for making additional MH-specific changes in the future.